<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTL SDR Spectrum Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="6,2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="1" column="1">
     <widget class="QToolBox" name="toolBox">
      <property name="minimumSize">
       <size>
        <width>255</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>255</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="toolTune">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>255</width>
         <height>423</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Tuner</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Center Frequency (Hz)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxFrequency">
          <property name="minimum">
           <number>20000000</number>
          </property>
          <property name="maximum">
           <number>2000000000</number>
          </property>
          <property name="singleStep">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Gain (dB)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxGain">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>38.6</string>
          </property>
          <property name="currentIndex">
           <number>21</number>
          </property>
          <item>
           <property name="text">
            <string>0.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0.9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7,7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16.6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22.9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29.7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32.8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>33.8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>36.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>37.2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38.6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>40.2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>42.1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>43.4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>43.9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>44.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>48.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>49.6</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>255</width>
         <height>423</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Spectrum</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Avaraging</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxAlfa">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="toolBoxLevel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>255</width>
         <height>423</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Waterfall</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="PlotWidget" name="spectrumView"/>
      <widget class="PlotWidget" name="waterfallView"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblFreq">
      <property name="text">
       <string>100.0Mhz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
